delivery_recipe/kick_starting_a_project.md

```diff
@@ -0,0 +1,64 @@
+# Kick Starting a Project
+
+Once a project has been sold the next step is usually an intense discovery process to map out processes, to look at possible solutions and to identify the high level requirements (epics). At the end of the discovery process the next step is to hand the client over to the project delivery team to begin the process of developing what’s been agreed.
+
+## Where to start
+
+It can be daunting when you first begin a project. The expectations are high, there is excitement among those who have been involved in the process so far and it’s your job to pick up the baton and see this through to the finish.
+
+The first step in the process of taking the project into the delivery phase is the handover from the discovery or sales team to the project delivery team. To maintain continuity and the trust of the client the handover will involve:
+
+* Introductions - introducing the client to the project delivery team
+* Project team - key contacts within the client
+* Epics - the key, high level requirements, initial priorities
+* Business and project goals - why are we doing this?
+* Budget - how much do we have to deliver with?
+* Schedule - what are the key dates?
+* Risks - known risks to manage, mitigation already identified
+* Reports - specific requirements for reporting
+
+It’s important to fully understand what you’re expected to deliver, why and who the users are before you delve into the delivery.
+
+## Developing client relationships
+
+Before anything starts it’s good to begin by sitting down with the Product Owner to talk through the project and what they can expect. There is no harm in going over some of the requirements that have already been discussed.
+
+You need to assert your authority early on by setting out the approach the team will take, how the epics will be prioritised, broken down into stories and then further down into specific tasks. Explain the product backlog, why this needs to be reviewed and updated regularly. Make sure the Product Owner understands Scrum and specifically what is required of them. This is your project, make sure you deliver it your way.
```

Comment:
rather than us 'asserting authority' it's more about setting the relationship off on an equal footing - they are the experts in their business, we are the experts in digital transformation. successfully delivery will only happen if we have equal status and voice.

And for the last sentence, this is the clients digital service, not ours - it's just that we're the ones with the experience of delivering. So it's our role to lead them through it and be the professional advisor, just as a surgeon would with a patient.